### DISCUSSION TITLE
Ability to adjust brightness of the inactive panes?

### DISCUSSION THREAD

**Discussion by briancoates** [Eugeny/tabby#9691] [Q&A] [answered]
I'd like to make the inactive panes a little more dimmer, so it stands out more.  Is that configurable?

**Comment by moemoechu** [comment, marked as answer]
try this custom css (1.0 means no dim):

split-tab>.child {
  opacity: 0.3;
}

**Reply by briancoates** [reply]
Thanks!  Where do i make that change?  I guess I was expecting to see a .tabby file in my home dir, but I don't.  (macOS)

**Comment by moemoechu** [comment]
in setting [Appearance] -> [Custom CSS] 
![image](https://github.com/Eugeny/tabby/assets/6707029/da389c3a-59d5-4324-991d-49727d282d43)

**Reply by briancoates** [reply]
🤦🏻‍♂️

thanks so much.
